tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md

```diff
@@ -82,7 +82,7 @@ Failed to connect to the TiDB cluster. It is recommended to check whether the Ti
 
 ### Error message: "TiDB cluster storage is not enough. Please increase the node storage of TiKV."
 
```

Comment:
This document can be removed because it is for the [Data Migration](https://docs.pingcap.com/tidbcloud/migrate-from-mysql-using-data-migration/) feature of TiDB Cloud Dedicated.